src/packages/web/page.gleam

```diff
@@ -76,6 +87,10 @@ fn package_list_item(package: PackageSummary) -> Node(t) {
       ),
       html.h2([], [external_link_text(url, package.name)]),
       html.p_text([], package.description),
+      case links {
+        [] -> html.span([], [])
```

Comment:
Is there a better way we can render nothing in the case that there are no links?

I had this idea, but I don't know if it's "idiomatic" Gleam:

```gleam
..{
  case links {
    [] -> []
    links -> [html.div([attrs.class("package-links")], links)]
  }
}
```

Reply:
I believe Nakai has a `html.Nothing` or similar.

Reply:
Indeed it does! Thanks.